src/cpp/rtps/RTPSDomain.cpp

```diff
@@ -192,7 +192,7 @@ RTPSParticipant* RTPSDomain::createParticipant(
                 PParam.builtin.metatrafficUnicastLocatorList))
     {
         // we do not log an error because the library may use participant creation as a trial for server existence
```

Comment:
The comment would become invalid, so it should be removed

Reply:
:+1: